### ISSUE TITLE
Stuck at "Restoring NuGets: WARNING: Error reading msbuild project information, ensure that your input solution or proje..."

### ISSUE THREAD

**Issue by felixding** [microsoft/react-native-windows#3946]
1. I followed the docs to start a new project:

```
choco install nodejs
yarn global add react-native-cli
react-native init rn --version 0.59.10
cd rn
yarn add rnpm-plugin-windows --dev
react-native windows
```

2. I installed VS 2019 Community

3. When I try to run the project with 

```
react-native run-windows
```

It says

```
Restoring NuGets: WARNING: Error reading msbuild project information, ensure that your input solution or proje...
```

I've attached the screenshot.

<img width="900" alt="Annotation 2020-01-24 085355" src="https://user-images.githubusercontent.com/111456/73036633-1ad69900-3e87-11ea-882d-60e460fd6839.png">

Can you please help? Thanks.

**Comment by licanhua** [comment]
I guess you didn't install v141(or possible SDK 10.0.18362.0), please check [system-requirements](https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/GettingStarted.md#system-requirements).

Your logs are not enough for troubleshooting, please provide logs of `react-native run-windows --logging`.
If you see `error MSB8036: The Windows SDK version 10.0.18362.0 was not found.`, please download and install it from https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk.

**Comment by felixding** [comment]
Okay. I don't know which part was missing but I re-installed everything. Now it is ` error MSB4019: The imported project "C:\Pr ogram Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v150\Microsoft.Cpp.Default.props" was not found.`

Why is it referencing VS Enterprise? Never have it installed.

<img width="900" alt="rn1" src="https://user-images.githubusercontent.com/111456/73585419-9ae2ab80-44e4-11ea-96b7-916be0b3ae38.png">
<img width="900" alt="rn2" src="https://user-images.githubusercontent.com/111456/73585424-9fa75f80-44e4-11ea-9c19-837acabc3626.png">

**Comment by felixding** [comment]
I manually updated `VCTargetsPath` in `./node_modules/react-native-windows/local-cli/runWindows/utils/msbuildtools.js` (the Enterprise path was hard-coded), and the previous issue was gone. Maybe it's related to https://github.com/microsoft/react-native-windows/issues/3430 ?

Unfortunately, now I have this:

<img width="900" alt="rn3" src="https://user-images.githubusercontent.com/111456/73587405-87dcd500-44fe-11ea-9574-9b7d91a88c5e.png">

**Comment by chrisglein** [comment]
@felixding We just noticed that you're creating a project against version 0.59, and not our "vnext". If you're standing up a new project now the default should be "vnext", and in fact we're working to firm up transition and retire "vcurrent" any day now. Since your project is new please try the steps for "vnext" to get you set up on our latest infrastructure: https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/GettingStarted.md
We'll be better able to support you there. Try that and let us know how it works. Or if you have a hard requirement for version 59 (vnext is on 60 now, adding support for 61 in a week or so, whereas current will end of life at 59).

**Comment by felixding** [comment]
@chrisglein 

I started with the `current` version as the docs says `next` is for prototyping and still under development, so I got the idea that `current` is, well, the current stable version.

I had manually updated some hard-coded settings (e.g path to VS) in `node_modules` and eventually made the demo project work.

Next I'll try to switch to `next` for my real world Windows app. Thanks for the help.
